apim/3.x/templates/api/api-configmap.yaml

```diff
@@ -129,6 +129,15 @@ data:
           username: {{ .Values.es.security.username }}
           password: {{ .Values.es.security.password }}
         {{- end }}
+        {{- if (eq .Values.es.lifecyle.enabled true) }}
+        lifecyle:
```

Comment:
I could be wrong, but I think ILM policies can only be configured from the gateway, and not from the management-api.

Reply:
That's an error. I'm correcting.

Reply:
It's done